src/components/AR/cameras/AR-Camera.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@wonderlandengine/api';
+
+abstract class ARCamera extends Component {
```

Comment:
I would use an `interface` instead if we do not store any data in `ARCamera`. Property chain lookup is slow.

Reply:
Someone might want to check `if(component instanceOf ARCamera)`which would not work with type of interface

Reply:
As long as it's not yet needed I wouldn't use it. Changing that later is relatively easy and compatible

Reply:
It is needed in the example where one can switch between different camera modes.